content/en/docs/concepts/traffic-management/index.md

```diff
@@ -261,8 +261,8 @@ subset named v2. You’ll see how you define a service subset in the section on
 Routing rules are **evaluated in sequential order from top to bottom**, with the
 first rule in the virtual service definition being given highest priority. In
 this case you want anything that doesn't match the first routing rule to go to a
-default destination, specified in the second rule. Because of this, the second
-rule has no match conditions and just directs traffic to the v3 subset.
+default destination, specify that in a second rule which has no match conditions.
+Here, the rule directs traffic to the v3 subset.
```

Comment:
A thought would be to have a snippet here with two rules, the match plus the default behavior if no match. Then replace this sentence with something like `Here, the second rule redirects traffic to the v3 subset if the first rule is unmatched.`

Reply:
I think the problem is that the example needs to be fixed, something like this:
```
{{< text yaml >}}
- match:
   - headers:
       end-user:
         exact: jason
  route:
  - destination:
      host: reviews
      subset: v2
- route:
  - destination:
      host: reviews
      subset: v3
{{< /text >}}
```
Then I would change the wording:
```
Routing rules are **evaluated in sequential order from top to bottom**, with the
first rule in the virtual service definition being given highest priority.
In the following example, anything that doesn't match the first routing rule
will go to a default destination as specified in the second rule, which has no match
conditions and just directs traffic to the v3 subset.
```

Reply:
In the original "In this case you want anything that doesn’t match the first routing rule to go to a default destination, specified in the second rule. ", it has been stated that anything that does not match the first routing rule goes to the default target specified in the second rule, so finally "if the first rule is unmatched" is not necessary.
<img width="735" alt="image" src="https://user-images.githubusercontent.com/67493131/196030271-3a1a6ae0-b597-4e66-9a9f-5f9c3e5a13c3.png">

Reply:
Speaking of examples, I want to know why v1 and v2 are not used in this example. I think this is more intuitive.

Reply:
In the "anything that doesn't match the first routing rule
will go to a default destination as specified in the second rule, which has no match
conditions and just directs traffic to the v3 subset.","which has no match
conditions and just directs traffic to the v3 subset",it is easy to see as a supplement to the default target rather than this example.

Reply:
I think in the "anything that doesn't match the first routing rule
will go to a default destination as specified in the second rule, which has no match
conditions and just directs traffic to the v3 subset.","which has no match
conditions and just directs traffic to the v3 subset",it is easy to see as a supplement to the default target rather than this example.

Reply:
The example is the unified style of the full text. I don't think there is much problem for the time being.